packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -189,7 +188,7 @@ const BpkInfoBannerInner = ({
       show={show}
       {...rest}
     >
-      <section className={sectionClassNames} role="alert">
+      <section className={sectionClassNames} role="presentation">
```

Comment:
`presentation` seemed like the most sensible value to me here as it removes it as a landmark which this probably. These banners are probably to be used as part of a section of the page and not contain a whole section themselves? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section